satpy/writers/__init__.py

```diff
@@ -322,6 +322,27 @@ def add_logo(orig, dc, img, logo=None):
     return XRImage(new_data)
 
 
+def add_scale(orig, dc, img, scale=None):
```

Comment:
I realize the other methods are done this way, but could `scale` be made a positional argument (no `=None`)? This would actually fail in the default case and based on the below if statements this will never be the case.

Reply:
@djhoese OK. Should we also delete `=None` in `add_logo` and `add_text`?

Reply:
If it is just as simple then I vote for yes. @mraspaud?

Reply:
I'm good with that